account_reconcile_oca/models/account_bank_statement_line.py

```diff
@@ -572,11 +572,15 @@ def _reconcile_data_by_model(self, data, reconcile_model, reconcile_auxiliary_id
             liquidity_amount += line_data["amount"]
             if line_data["kind"] == "liquidity":
                 default_name = line_data["name"]
+        partner = (
+            reconcile_model._get_partner_from_mapping(self) or self._retrieve_partner()
+        )
         for line in reconcile_model._get_write_off_move_lines_dict(
-            -liquidity_amount, self._retrieve_partner().id
+            -liquidity_amount, partner.id
         ):
             new_line = line.copy()
             new_line["name"] = new_line.get("name") or default_name
+            new_line["partner_id"] = partner and partner.name_get()[0] or False
```

Comment:
`partner_id` contains a string? Isn't the key name misleading?

Reply:
This vals are reused on the widget, not for the creation. it is transformed later. it is true that partner_id might not be the best name, but this is what we have right now.

Reply:
You can add a previous refactoring commit renaming all its uses for not continuing spreading an incorrect value.